receiver.go

```diff
@@ -80,6 +81,17 @@ func (r *Receiver) HandleMessage(ctx context.Context, handle func(*Message) erro
 	}
 }
 
+// AddCredit adds credits to the current link.
+func (r *Receiver) AddCredit(credit uint32) error {
+	return r.link.addCredit(credit)
+}
+
+// Drain sets the drain flag on the next flow frame and
+// waits for the drain to be acknowledged.
+func (r *Receiver) Drain(ctx context.Context) error {
```

Comment:
DrainCredits to match IssueCredits (same pattern as .NET)?

Reply:
Makes sense. Will change.

Reply:
Fixed. There were a few complementary spots I also had to change but it should now be consistently either IssueCredit or DrainCredit